### ISSUE TITLE
The data from metaData variable does not appear in the "reconstruct table from model" window 

### ISSUE THREAD

**Issue by mmaunier** [texstudio-org/texstudio#1585]
### Environment
-  TeXstudio: AppImage (from [texstudio.org](https://github.com/texstudio-org/texstudio/releases/download/3.1.1/texstudio-3.1.1-x86_64.AppImage))
-  OS: GNU/Linux (Manjaro ; kernel 5.10)
-  KDE : 5.21.3
-  Qt : 5.15.2

### Expected behaviour
When creating table template file .js in `$HOME/.config/texstudio`,  we can use it in menu : LaTeX -> table manipulation -> reconstruct table from model 
Then, a window is showing up with user template list and metadata should be collected and visible in the center of the window.

### Actual behaviour
Even with model  `tabletemplate_fullyframed_firstBold.js` , nothing is showing in the window (see attachments). 
- Name of template is not showing in the menu at top left (it's blank), but script is well created.
- When selecting the blank template name, window center shows no information (no name, <no description>, <no author> ... etc)
- But, the template works fine when validated (I presume it's a problem when getting resource from  metadata variable)
- I can put a `.png` file with same name as `.js` file in folder `$HOME/.config/texstudio` and the picture is showing at right of the template window.
- If I do not put the `var metaData = {` on first line of `.js` file, but something else, i get appearing the name of the `.js` file instead, in the menu of window template (no other informations in the center of window, on the other hand). 

### How to reproduce  
When putting a .js file in `$HOME/.config/texstudio`, i get the same behaviour. With multiple table templates, i get multiple blank lines in window menu.

### Attachment 
![image](https://user-images.githubusercontent.com/2765724/115111222-879fec80-9f7f-11eb-9d2f-54f617ed3841.png)
![image](https://user-images.githubusercontent.com/2765724/115111251-b1f1aa00-9f7f-11eb-8754-5f4dcf823bb1.png)

**Comment by mmaunier** [comment]
Good evening,
thanks for the patch. 
I have downloaded the new AppImage (file version attached) but unfortunately there is still the same behaviour.
I have the same window with no information.
M. Maunier

![image](https://user-images.githubusercontent.com/2765724/115304745-9a741600-a165-11eb-8bd9-0fb86aacf44b.png)

**Comment by sunderme** [comment]
I have checked with the latest dev build and there it works

**Comment by mmaunier** [comment]
I installed this build too, but no chance, this doesn't work either... Same behavior.
(texstudio-linux-3.1.2-202104191705-git_366901e-x86_64_49063dd71a79ddb8f5d6904746043ddf.AppImage)
Is there a way I can help in order to help you finding what's going wrong ? (a debug mode or something else ?)
Thanks.

**Comment by sunderme** [comment]
i would suggest to build locally

**Comment by mmaunier** [comment]
Ok, I compiled it from the git repository. Unfortunately, I still had the same problem, but we were moving forward ...
The builtin template files appeared so I went to see `/usr/share/texstudio/`, and they were fine there.
I take this opportunity to tell you that the date does not appear correctly on the other hand (even on the texstudio templates)

So I copied my personal file in this directory to test ... and there indeed, it worked correctly. (see attachments)

So the problem was that the custom files in the `$HOME/.config/texstudio/` directory did not work.
So I deleted it, relaunched TeXStudio, put it back and there, miracle, it worked ... (without the date).

Thank you for the help !
Cordially. 

![Screenshot_20210421_125327](https://user-images.githubusercontent.com/2765724/115544994-93ebb880-a2a3-11eb-88bf-0437a9f73563.png)
![Screenshot_20210421_125203](https://user-images.githubusercontent.com/2765724/115544999-951ce580-a2a3-11eb-81ae-5c9e1ef31fe7.png)
